ArcadiaParties-SPA/src/app/services/unauthorized-interceptor.ts

```diff
@@ -15,20 +15,15 @@ import { AuthService } from './auth.service';
 
 @Injectable()
 export class UnauthorizedInterceptor implements HttpInterceptor {
-  constructor(private authService: AuthService) {}
+  constructor(private authService: AuthService) { }
 
   intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
     return next.handle(request).pipe(
       tap(
-        (event: HttpEvent<any>) => {
-          if (event instanceof HttpResponse) {
-          }
-        },
+        null,
```

Comment:
Use `() => {}` instead.

Reply:
Otherwise, you could use `catchError` pipe method to catch all errors.

Reply:
`tap` is deprecated. I've implemented `catchError`